downstream/modules/hub/proc-install-ha-hub-selinux.adoc

```diff
@@ -44,9 +49,10 @@ $ mount -a
 $ setup.sh -- -b --become-user root
 ----
 
+.Next steps
 Once the installation is complete, unmount the `/var/lib/pulp/` mount point then apply the appropriate SELinux context:
```

Comment:
Change to "After the installation is complete ..." Only use once to indicate "one time" and should not be used to mean after or when.